cmake/sdk_cmake_targets_readme.md

```diff
@@ -66,21 +66,30 @@ structure example.
 ### Creating an unsigned enclave
 
 Enclave targets in CMake should use `add_executable`, link to
-`openenclave::oeenclave`, and include the trusted EDL code if `oeedger8r` was
-used. An example is:
+`openenclave::oeenclave` and `openenclave::oelibc`, and include the trusted EDL
+code if `oeedger8r` was used. An example is:
 
 ```cmake
 add_executable(example_enclave example.c example_t.c)
-target_link_libraries(example_enclave openenclave::oeenclave)
+target_link_libraries(example_enclave openenclave::oeenclave openenclave::oelibc)
 ```
 
+Technically, the `openenclave::oelibc` dependency is optional. That is, if you
+don't use anything from the C library (such as `#include <stdio.h>` etc.), you
+can exclude it, but this is very unlikely.
+
 If the enclave uses C++ code, you also need to link `openenclave::oelibcxx`:
```

Comment:
Consider "... you should link `openenclave::oenclave` and `openenclave::oelibcxx`." which reads less like a contradiction to your explanation after the sample.

Reply:
Thanks!